src/parser/identifier-parser.ts

```diff
@@ -16,6 +16,14 @@ export function parseSchemableIdentifier(id: string): SchemableIdentifierNode {
   }
 }
 
+export function parseSchemableIdentifierArray(id: string | string[]): SchemableIdentifierNode[] {
+  if (Array.isArray(id)) {
+		return id.map(parseSchemableIdentifier)
+	} else {
+		return [parseSchemableIdentifier(id)]
+	}
```

Comment:
Let's simplify this a bit.

```suggestion
  if (!Array.isArray(id)) {
    id = [id]
  }
  
  return id.map(parseSchemableIdentifier)
```